backend/routes/api/card_management.py

```diff
@@ -567,6 +569,206 @@ def flashcard_exists():
+        
+#         #find all term-definition matches from parsed data
+
+#         matches = re.findall(QUIZLET, str(soup), re.DOTALL)
+#         flashcards = {}
+
+#         cards = []
+#         #create cards based on term-definition data
+#         for term, definition in matches:
+#             definition = definition.replace('<br/>', '').replace('<br>', '').replace('<br />', '')
+#             cards.append({
+#                 "front": term,
+#                 "back": definition
+#             })
+#             flashcards[term] = definition
+
+#         flashcard_id = hash_to_numeric(user_id + folder + flashcard_name)
+        
+#         flashcard_exists = db.folders.flashcard_exists(user_id, flashcard_id)
+
+#         if flashcard_exists:
+#             return jsonify({"error": "Flashcard set name already exists"}), 400
+
+#         # Generate the card_ids
+#         card_ids = [
+#             hash_to_numeric(user_id + folder + flashcard_name + card["front"])
+#             for card in cards
+#         ]
+   
+#         # Create the flashcard set
+#         db.flashcard_set.create_flashcard_set(
+#             flashcard_id=flashcard_id,
+#             flashcard_name=flashcard_name,
+#             flashcard_description=flashcard_description,
+#             card_ids=card_ids,
+#             user_id=user_id,
+#         )
+
+#         # Create each individual flashcard
+#         db.flashcards.create_flashcards(card_ids, cards)
+
+#         # Assign the set to the user in the folder structure
+#         db.folders.add_flashcard_to_folder(
+#             user_id=user_id,
+#             folder=folder,
+#             flashcard_id=flashcard_id,
+#             flashcard_name=flashcard_name,
+#             card_ids=card_ids,
+#         )
+
+#         # Give the user read and write access
+#         db.read_write_access.give_user_access(user_id, flashcard_id)
+
+#         return jsonify({"success": "Flashcards imported from Quizlet"}), 200
+#     except Exception as e:
+#         return jsonify(str(e)), 500
+
+@card_management_routes.route("/api/import-from-quizlet", methods=["POST"])
+@validate_json(IMPORT_FROM_QUIZLET_FORMAT)
+def import_from_quizlet():
+    
+    try:
+        user_id = request.json.get("userID")
+        folder = request.json.get("folder")
+        flashcards_string = request.json.get("flashcards")
+        term_def_separator = request.json.get("term_def_separator")
+        term_separator = request.json.get("term_separator")
+        flashcard_name = request.json.get("flashcard_name")
+        folders =  db.folders.get_user_data(user_id)
+        
+        # Assuming this method exists
+        if not folders:
+            return jsonify({"error": "Folder does not exist" + folder}), 404
+        found = None
+
+        #checking if folder exist
+        if folder not in folders:
+            return jsonify({"error": f"Folder '{folder}' does not exist"}), 404
```

Comment:
Please can you delete lines `695` to `702`? So that if the folder doesn't exist, it'll be created anyway? (This is how the other API routes work, and it also means that you can set the folder as `""` to not save it in any folder. I've tested it locally to make sure it works)

Once you've done that (and updated any unit tests if needed), I'll approve and merge

Thanks!